content/docs/install.md

```diff
@@ -13,75 +13,91 @@ Odin is dead-simple to get started with!
 ## Clone or download Odin binaries
 You can either:
 
-Clone the repository `git clone https://github.com/odin-lang/Odin` (recommended).
+- Clone the repository: `git clone https://github.com/odin-lang/Odin` (makes [updating](#updating-the-compiler) the compiler easier)
 
-Or download the latest binaries and add them to your path:
+or download the latest binaries and add them to your path:
 
-Download [the latest release](https://github.com/odin-lang/Odin/releases/latest).
+- Download [the latest release](https://github.com/odin-lang/Odin/releases/latest)
+- Download [the latest nightly build](/docs/nightly/)
 
-Download [the latest nightly build](/docs/nightly/).
+**Note**: If you use the binary version, then you can skip the "Building Odin" section below, but the stuff under [requirements](#requirements) is still required!
 
-**Note**: Cloning the repository is recommended in order to make [updating](#updating-the-compiler) easier.
+### Platform and CPU support 
+Odin supports x86-64/AMD64 on Windows, Linux and macOS, and ARM64 on macOS.
 
-### Support 
-Odin supports x86-64/AMD64 on Windows, Linux and macOS, and ARM64 on macOS. Odin also relies on LLVM (for code generation) and an external linker.
+Additional supported platforms are FreeBSD, OpenBSD, NetBSD and Haiku. There are no binary releases for those platforms.
 
 ### Requirements
-The following platform-specific steps are necessary:
+Odin relies on LLVM (for code generation) and an external linker. These are needed regardless of if you build from source or use the binary releases. When building from source you also need a C++ compiler. Here follows platform-specific steps on how to acquire these dependencies:
 
-- Windows
-    * Have Visual Studio installed (VS2019-2022 is recommend, VS2017 will likely work, for the linker)
-    * Open a valid command prompt:
-        * **Basic:** run the `x64 Native Tools Command Prompt for VS2017` shortcut bundled with VS 2017, or
-        * **Advanced:** run `vcvarsall.bat x64` from a blank `cmd` session
-
-- MacOS
+* Windows
+    * Install Visual Studio 2019-2022 (2017 may also work)
+    * Install Visual Studio workload "Desktop development with C++", required components from that workload are the MSVC build tools and the Windows SDK
+    * No LLVM installation is needed, the LLVM DLL is bundled in the repository
+* MacOS
     * Install the latest XCode (from the App Store or the [Xcode website](https://developer.apple.com/xcode/))
     * Install XCode command-line tools `xcode-select --install`
     * Install [Homebrew](https://brew.sh/) 
     * Install LLVM through Homebrew with: `brew install llvm@14`
```

Comment:
MacOS doesn't need brew or LLVM when using a release

Reply:
To be fair it is probably fine like this, this page is in dire need of an update, it mentions a bunch of outdated things that wasn't the aim of this pr

Reply:
I see, perhaps the whole page should be split into "from release/from source". It would probably make both options simpler.